NuGet.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
OK, thanks for clarifying. I wondered if it was strictly necessary in some situation. If a scenario occurs where this is needed then we can add it. Until then I'm trying to minimise the amount of boilerplate stuff.